### PR TITLE
Issue #1780 - Letter count #DWOC

### PR DESCRIPTION
**Pull request by shloka-gupta**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

I did
### Description

I added the letter count program

### Checklist

- [Yes ] I've read the contribution guidelines.
- [Yes] I've checked the issue list before deciding what to submit.
- [Yes] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

issue #1780

### CODE REVIEW

**Comment by shloka-gupta** [comment]
Added Letter Count program from issue #1780 . Let me know if there are any changes required

**Review by atarax665 — CHANGES_REQUESTED** [review]
Format your code according to PEP8 guidelines. You can use [this](http://pep8online.com/) to format your code.

**Comment by shloka-gupta** [comment]
I did so

File: Python/cp/LetterCount.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            dict[c] = co + 1
+
+    return(dict)
+
+
+word = input("enter a word ").lower()
+
+LetterCount = LetterCount(word)
+
+print(LetterCount)
```
Comment (HarshCasper) [inline Python/cp/LetterCount.py]: Enclose in a `if __name__ == '__main__'` block

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: Python/cp/LetterCount.py (hunk 1 above)
Comment (shloka-gupta) [reply Python/cp/LetterCount.py]: Done @HarshCasper

**Comment by shloka-gupta** [comment]
Kindly, re-review

File: Python/cp/LetterCount.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            dict[c] = co + 1
+
+    return(dict)
+
+if __name__ == '__main__':
+    word = input("enter a word ").lower()
+    LetterCount = LetterCount(word)
+    print(LetterCount)
+
+else:
```
Comment (atarax665) [inline Python/cp/LetterCount.py]: I don't think we need an else statement here.

**Review by atarax665 — CHANGES_REQUESTED** [review]

File: Python/cp/LetterCount.py (hunk 2 above)
Comment (HarshCasper) [reply Python/cp/LetterCount.py]: @chicken-biryani This looks like an Anti-Pattern

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: Python/cp/LetterCount.py (hunk 2 above)
Comment (shloka-gupta) [reply Python/cp/LetterCount.py]: @atarax665 @HarshCasper Apologies. Part of learning I guess (Thank you for the new word, Anti-pattern).

**Review by atarax665 — APPROVED** [review]

**Pull request merged**